resource-watch/resource-watch
# removes deprecated widget-editor assets

## Overview
Removes "help" images used in `widget-editor@1.x`. No need to keep them with `widget-editor@2.x`


## Testing instructions
Run the application normally, you should not find any issue. The Widget Editor should work as usual as this change shouldn't affect the current version.

The below message should not appear anymore when the app initializes.

![image](https://user-images.githubusercontent.com/999124/94028309-12d74c00-fdbc-11ea-9e32-159c55ac0686.png)

## Pivotal task
https://www.pivotaltracker.com/story/show/174952659

---

## Checklist before submitting
